cognite/neat/_rules/models/entities/_wrapped.py

```diff
@@ -193,7 +193,7 @@ class RawFilter(DMSFilter):
     filter: str
     inner: None = None  # type: ignore[assignment]
 
-    def as_dms_filter(self) -> dm.Filter:  # type: ignore[override]
+    def as_dms_filter(self, default: Any | None = None) -> dm.Filter:  # type: ignore[override]
```

Comment:
Unrelated to the bugfix.. This was just an inconsistency, the base class `DMSFilter.as_dms_filter(...)` defines this method with this parameter.

Reply:
Not for now maybe, but we should try to avoid "type: ignore".